src/unicef_djangolib/etag.py

```diff
@@ -1,54 +0,0 @@
-import uuid
-from functools import wraps
-
-from django.core.cache import cache
-from django.db import connection
-from django.utils.cache import patch_cache_control
-from rest_framework import status
-from rest_framework.response import Response
-
-
-def etag_cached(cache_key, public_cache=False):
```

Comment:
why are we removing this code? This can be reused in other places.

Reply:
the etag should not be in djangolib but in the restlib